lib/msf/base/serializer/readable_text.rb

```diff
@@ -51,33 +51,55 @@ def self.dump_module(mod, indent = "  ")
   # @return [String] the string form of the table.
   def self.dump_exploit_targets(mod, indent = '', h = nil)
     tbl = Rex::Text::Table.new(
-      'Indent'  => indent.length,
       'Header'  => h,
       'Columns' =>
         [
+          '',
```

Comment:
Might be worth adding a comment saying what this is for since it's not immediately obvious

Reply:
Will get a comment added there.